src/main/java/uk/gov/hmcts/probate/service/exceptionrecord/mapper/ExceptionRecordGrantOfRepresentationMapper.java

```diff
@@ -251,11 +254,8 @@ public interface ExceptionRecordGrantOfRepresentationMapper {
         ToIHTFormCompletedOnline.class})
     @Mapping(target = "ihtReferenceNumber", source = "ocrFields.ihtReferenceNumber")
     @Mapping(target = "ihtFormId", source = "ocrFields", qualifiedBy = {ToIHTFormId.class})
-    @Mapping(target = "ihtGrossValue", expression = "java(OCRFieldIhtMoneyMapper.poundsToPennies(new String("
-            + "\"ihtGrossValue\"), ocrFields.getIhtGrossValue()))", qualifiedBy = {ToPennies.class})
-    @Mapping(target = "ihtNetValue", expression = "java(OCRFieldIhtMoneyMapper.poundsToPennies(new String("
-            + "\"ihtNetValue\"), ocrFields.getIhtNetValue()))", qualifiedBy = {ToPennies.class})
-
```

Comment:
if we are used to this pennies conversion is it going to be confusing for caseworker if new equivalent values are in pound?